src/components/Card/Card.js

```diff
@@ -11,7 +12,22 @@ class Card extends React.Component {
         <h3 className="p-card__title">{this.props.title}</h3>
         <p className="p-card__content">{this.props.cardContent}</p>
       </div>
-    )
+    );
   }
 }
-export default Card;
\ No newline at end of file
+
+Card.defaultProps = {
+  modifier: '',
+  imgSrc: 'http://placehold.it/200x32',
+  imgAlt: 'Placeholder',
+};
+
+Card.propTypes = {
+  modifier: PropTypes.string,
+  imgSrc: PropTypes.string,
+  imgAlt: PropTypes.string,
+  title: PropTypes.string.isRequired,
+  cardContent: PropTypes.string.isRequired,
```

Comment:
If you change `props.cardContent` as I suggested before either here, or in another component, you can set this to `PropTypes.node`.

If you decide to use `{img: {src: '', alt: ''}}` as I also suggested you can do:
```
img: {
  src: PropTypes.string,
  img: PropTypes.string
}
```